lms/static/sass/views/_teams.scss

```diff
@@ -526,7 +530,23 @@
         }
     }
 
-    .form-actions {
+    .required-wrapper {
+      display: inline-block;
+      vertical-align: top;
+      width: 60%; // TODO: susy grid
+    }
+
+    .optional-wrapper {
+      display: inline-block;
+      vertical-align: top;
+      width: 35%; // TODO: susy grid
+      margin-left: 2%;
```

Comment:
For left / right rules, don't forget to wrap them to account for RTL styling. 

Here's a quick reference document for this: https://openedx.atlassian.net/wiki/display/UX/RTL+Sass+Cheatsheet

example: @include margin-left(x); would make this RTL friendly.